examples/asr/speech_to_text_rnnt.py

```diff
@@ -74,6 +90,13 @@ def main(cfg):
     exp_manager(trainer, cfg.get("exp_manager", None))
     asr_model = EncDecRNNTModel(cfg=cfg.model, trainer=trainer)
 
+    # initialize the model with the weights of the checkpoint specified by 'load_weights_from_checkpoint' in the configs
+    # You may use this option to start the training from a pre-trained checkpoint
+    checkpoint_path = cfg.model.get("load_weights_from_checkpoint", None)
```

Comment:
Same comments as below - pop out this command line override, load checkpoint in seperate line, load checkpoint into model and explicitly del the entire checkpoint. 

X = torch.load()
Model.load_state_dict(X[state_dict])
del X